### PR TITLE
Add blogpost from Berlin

### PR DESCRIPTION
**Pull request by flexbox**

### CODE REVIEW

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+date: 2016-07-06
+author: David
+page_title: "I visit / work / live in the contentful HQ during 2 days"
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I visited / worked / lived in the HQ of Contentful for 2 days

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+title: "Berlin"
+description: "One day at Contentful"
+cover_picture: berlin.jpg
+banner_picture: tv-tower.jpg
+---
+
+Guten Tag! it’s David!<br>
+Actually I work as a [Front-end engineer consultant](http://www.davidl.fr/) in Lille, France. I have the chance to visit the [contentful HQ](https://www.contentful.com/about-us/) in Berlin.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I currently work as a ...

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I had the chance to visit ...

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+4 Days to explore this city, meet the team, plus an amazing mission for me at the end, see the match France vs Germany in incognito mode.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: for me in the end: watch the match ...

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+READMORE
+
+## Berlin — Capital of Germany
+
+During my trip, I visited a lot of places only by walking. It would be a shame to not enjoy the sun, and it’s easier to practive my favorite activity : __getting lost & dicover new places__.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: [..] of places just by walking.

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: It would have been a shame ...

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: the sun and ... 
(no comma after `sun`)

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: to practice ...

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: and discovering ...

File: source/blog/articles/2016-07-06-contentful-berlin.md (hunk 3 above)
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: watch the match ...

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+- Alexander Platz
+- Berliner Dom
+- The chaos Computer club
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: Capital letter on each word: `The Chaos Computer Club`

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+- Checkpoint Charlie
+- Plenty of “straße”
+- & Ampelmann Shop (Yes, in Berlin there are custom traffic light man)
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: men

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+![David Leuliette signature in Ampelmann Shop Berlin](/assets/images/blog/articles/2016-07-06-contentful-berlin/ampelmann-signature.jpg)<br>
+_A small dedicace on the wall because I am famous_
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I'm

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+
+### Agile Board Hacks at ThoughtWorks
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I'd use only 2 `#`.
Proposal: `Meeting people: Agile Board Hacks at ThoughtWorks`

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+This was my first time in Berlin. So what is the best way to meet some people ? … I am a community Leader on meetups in Lille. Maybe I can find a great one in Berlin!
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: a community leader of some meetups in Lille

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+
+Few hours after landing, I am at ThoughtWorks Werkstatt for an event called [Agile Board Hacks](http://www.meetup.com/fr-FR/ThoughtWorks-Berlin/events/232224931/).
+
+Nick Thorpe use examples of agile walls from his blog — [https://agileboardhacks.com](https://agileboardhacks.com) — where teams try to ship features faster, or solve problems.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: uses

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: try to ship features faster or try to solve problems.

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+
+![Agile board hack sketchnote form meetup in Berlin](https://c2.staticflickr.com/8/7356/27515657563_8c119c6d69_z.jpg)
+![Agile board hack sketchnote form meetup in Berlin](https://c2.staticflickr.com/8/7466/28027873082_7509f5351a_z.jpg)
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: Hum.. Try to find a different alt text.

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+
+_If you want sketchnotes for your next trip [join us as a rider](/riders)_
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md:41]: :+1: :smile:

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+
+## about:contentful
+
+[Contentful](https://www.contentful.com/) have a mission : __avoid the web-focused CMSs__ — aka See As Mess systems. Those didn’t fit for the web in XXI century — the web of mobile, interactive & multichannel content.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: has a mission

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 14]
```diff
@@ -0,0 +1,98 @@
...
+
+Actually contentful is a tool for developer : an API with a focus on content.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: for developers: it's an API

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 15]
```diff
@@ -0,0 +1,98 @@
...
+
+- Developers are happy because it’s not a [bloated backend](https://en.wikipedia.org/wiki/Software_bloat)
+- CMS addicts are happy because they can edit content on a web platform
+- Business is happy because they have a [clear pricing page](https://www.contentful.com/pricing/)
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: Businesses are happy

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 16]
```diff
@@ -0,0 +1,98 @@
...
+
+![contentful office in berlin](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-hq-berlin.jpg)
+
+_This is where magic happens_
+
+I talked with la lot of smart people who are involved in their product.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: involved in the product

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 17]
```diff
@@ -0,0 +1,98 @@
...
+
+As a Chief Marketing Officer, Chris want multiple experiences for Contentful. What if customer were allowed to have built-in solution ? You push the famous big red button and — Bam! Your new website is online with an astonish theme… Maybe they are working on it ? #revelation #stayTunned
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: wants

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: astonishing theme

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 18]
```diff
@@ -0,0 +1,98 @@
...
+
+Artas is Product Marketing Manager and one of his issue is about about __automation__. It’s funny because it remembers me one of my favorite quote.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: his issues

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: it reminds me

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: favorite quotes

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 19]
```diff
@@ -0,0 +1,98 @@
...
+
+> Whatever you do, make sure you are __automating__, because if you aren’t, all you are doing is making it harder for yourself when you revisit the __problem__, and making it even harder for the next __person__ who will inevitably have to __reproduce__ your __solution__.
+> <cite> Githuber at Puppet conf</cite>
+
+I encounted Eugene — Copywriter & he share me an amazing website on Typography [typesetinthefuture.com](https://typesetinthefuture.com/).
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I also met

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: he shared with me

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: website about Typography

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 20]
```diff
@@ -0,0 +1,98 @@
...
+
+But wait it's lunch time!
+![Guten appetit](/assets/images/blog/articles/2016-07-06-contentful-berlin/team-building-meal.jpg)
+_Homemade lemonade & mushroom salad_
+
+After the lunch, I meet Marko, Claus & Jan (Designer & developer).
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I met

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+
+We talked about how I code / organise atomic design systems like [patternlab](patternlab). We are all obsessed with performance and we share our best tools about image optimsation :
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: Can you provide a link to patternlab?

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: performance so we shared with each other our best tools

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 22]
```diff
@@ -0,0 +1,98 @@
...
+
+- [svgo](https://github.com/svg/svgo) : A nodejs-based tool for optimizing SVG vector graphics files.
+
+- [ImageOptim-CLI](https://github.com/JamieMason/ImageOptim-CLI) : Automate the optimisations of images.
+
+At the end of the day, I have a great time with the Kr3w : playing table tennis in the chilling zone.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: I had

File: source/blog/articles/2016-07-06-contentful-berlin.md (hunk 8 above)
Comment (flexbox) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: Great!

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 23]
```diff
@@ -0,0 +1,98 @@
...
+
+![Contentful chilling zone](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-chilling-zone.jpg)
+_Basement chilling zone!_
+
+Before leave the team they offer me an amazing gift : the __goodies box__! What’s include inside ?
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: Before leaving

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: they offered

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: What does it include?

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 24]
```diff
@@ -0,0 +1,98 @@
...
+
+- A color book
+- Rubicube
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: A Rubik's Cube

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 25]
```diff
@@ -0,0 +1,98 @@
...
+- Stickers
+- Tshirt
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: A Tshirt

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 26]
```diff
@@ -0,0 +1,98 @@
...
+At the end of the day, I have a great time with the Kr3w : playing table tennis in the chilling zone.
+
+![Contentful chilling zone](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-chilling-zone.jpg)
+_Basement chilling zone!_
+
+Before leave the team they offer me an amazing gift : the __goodies box__! What’s include inside ?
+
+- A color book
+- Rubicube
+- Stickers
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: A bunch of stickers

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 27]
```diff
@@ -0,0 +1,98 @@
...
+- Tshirt
+- and … ? and … ? and … ? Socks!
+
+![contentful goodies with T-shirt & Socks](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-goodies.jpg)<br>
+_Tweet about the socks #contentsocks @contenful_
+
+## What next?
+
+If you like ruby, [middleman](https://middlemanapp.com/) and want to dig on contentful [follow me on twitter](https://twitter.com/_flexbox)! I will soon have a surprise for you :)
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md:104]: Please also finish by talking about "rides" :wink: 
eg. try to answer to these questions:
- what do you think about visiting new cities both as tourist and tech guy/worker?
- will you try to go on remote working with someone the next time?
- etc.

**Comment by cveneziani** [comment]
General feedbacks:
- no space before `?`
- no space before `:`
- no coma before `and` / `or`

**Comment by cveneziani** [comment]
Toc proposal:
- Berlin — Capital of Germany
- Meeting people: Agile Board Hacks at ThoughtWorks
- 2 days at Contentful HQ
  - about:contenful
  - Meeting the team
  - Time to chill a bit
- What next?

`Meeting the team` would be just after `This is where magic happens`.
`Time to chill a bit` would be just after `Automate the optimisations of images`

What do you think?

**Comment by cveneziani** [comment]
Review done.

**Comment by flexbox** [comment]
Fix done!

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 28]
```diff
@@ -0,0 +1,107 @@
...
+
+_This is where magic happens_
+
+### Meeting the team
+
+I talked with la lot of smart people who are involved in the product.
+
+As a Chief Marketing Officer, Chris wants multiple experiences for Contentful. What if customer were allowed to have built-in solution? You push the famous big red button and — Bam! Your new website is online with an astonishing theme… Maybe they are working on it? #revelation #stayTunned
+
+Artas is Product Marketing Manager and one of his issues is about about __automation__. It’s funny because it reminds me one of my favorite quote.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: about about (duplicate)

Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: favorite quotes (plural form for the quotes)

File: source/blog/articles/2016-07-06-contentful-berlin.md [hunk 29]
```diff
@@ -0,0 +1,107 @@
...
+- and …? and …? and …? Socks!
+
+![contentful goodies with T-shirt & Socks](/assets/images/blog/articles/2016-07-06-contentful-berlin/contentful-goodies.jpg)<br>
+_Tweet about the socks #contentsocks @contenful_
+
+## What next?
+
+If you like ruby, [middleman](https://middlemanapp.com/) and want to dig on contentful [follow me on twitter](https://twitter.com/_flexbox)! I will soon have a surprise for you :)
+
+Berlin is an amazing city, if you want to joins our next ride feel free to contact us! We will go to the [coworking betahaus](http://www.betahaus.com/berlin/) & after a working day, we gonna eat some currywurst.
```
Comment (cveneziani) [inline source/blog/articles/2016-07-06-contentful-berlin.md]: to join (singular)

**Comment by flexbox** [comment]
fix done 🚀

**Comment by cveneziani** [comment]
Yeah! I'll try to merge it :smile:

**Pull request merged**